.github/workflows/main.yml

```diff
@@ -0,0 +1,37 @@
+name: Docurium
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby: [head, 2.6, 2.5, 2.4]
+        os: [ ubuntu-18.04, macos-latest ]
+
+    name: Ruby ${{ matrix.ruby }} on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ endsWith(matrix.ruby, 'head') || matrix.ruby == 'debug' }}
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Linux packages
+        if: runner.os == 'Linux'
+        run: |
+          sudo apt update
+          sudo apt install -y libclang-6.0-dev llvm-6.0 clang-6.0
```

Comment:
This is fine for now but we should also have some tests run with a different version of LLVM. This was only really set to 6 due to some bugs that meant docurium was often broken with different versions.

Reply:
I had a hunch compatibility was the issue. I'll investigate how easy it is to move LLVM version as a matrix parameter.